ui-kit/src/components/Tabs/TabNode.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { Tab } from './types';
+
+export interface TabNodeProps {
+  id: string;
+  tab: Tab;
+  active: boolean;
+  tabWidth?: number;
+  onClick?: (e: React.MouseEvent) => void;
+  tabBarGutter?: number;
+  onFocus: React.FocusEventHandler;
+}
+
+function TabNode(
+  {
+    id,
+    active,
+    tab: { key, tab, disabled },
+    tabBarGutter,
+    tabWidth,
+    onClick,
+    onFocus,
+  }: TabNodeProps,
+  ref: React.Ref<HTMLDivElement>
+) {
+  const tabPrefix = 'lubycon-tab';
+
+  const nodeStyle: React.CSSProperties = { marginLeft: tabBarGutter, width: tabWidth };
+
+  function onInternalClick(e: React.MouseEvent) {
+    if (disabled || !onClick) return;
+
+    onClick(e);
```

Comment:
```suggestion
    onClick?.(e);
```
으로 하면 `if` 문 안에 있는 `!onClick` 조건을 제거할  수 있을 것  같습니다

Reply:
반영해두었습니당